src/indexers/indexer.utils.ts

```diff
@@ -105,31 +101,6 @@ export async function constructVaultDefinition(
   });
 }
 
-export async function getLpTokenBalances(chain: Chain, vault: VaultDefinitionModel): Promise<VaultTokenBalance> {
```

Comment:
has this been deprecated? its not used?

Reply:
it is used in `vault-balances-indexer.ts`